db/postgres.go

```diff
@@ -44,6 +50,7 @@ type PostgreSQL struct {
 	KeyFieldName          string
 	ValueFieldName        string
 	PartnersJSONFieldName string
+	ExtraIndexesFields    ExtraIndexes
```

Comment:
Esse campo precisa ter informações de um ou mais índices (e o nome do campo não precisa do sufixo `Field`):

```suggestion
	ExtraIndexes    []ExtraIndex
```

Reply:
Aqui no caso nao posso usar ExtraIndexes porque ja há uma funcao com esse nome. Vou manter o nome atual porem botar o slice

Reply:
Não, lembra dos nomes curtos? Mude para `Indexes` então, mas não deixe o sufixo. Não é necessário e não adicona nada em termos de semântica chamar uma campo de _campo_.

Reply:
O problema e exatamente esse.. ja existe uma funcao chamada ExtraIndexes:

func (p *PostgreSQL) ExtraIndexes(idxs []string) error 

Mudo o nome dessa funcao para manter o ExtraIndexes entao?